src/AutoPilotPlugins/Common/RadioComponentController.cc

```diff
@@ -867,9 +867,7 @@ void RadioComponentController::_loadSettings(void)
     _transmitterMode = settings.value(_settingsKeyTransmitterMode, 2).toInt();
     settings.endGroup();
 
-    if (_transmitterMode != 1 || _transmitterMode != 2) {
-        _transmitterMode = 2;
-    }
+    _transmitterMode = 2;
```

Comment:
Not sure why you think this is not needed. The current code is correct and required to validate the two valid settings from _transmitterMode.

Reply:
`if (_transmitterMode != 1 || _transmitterMode != 2)`
This is always true.

Reply:
Ah, right it's screwed up. Should be `if (!(_transmitterMode == 1 || _transmitterMode == 2))`

Reply:
@DonLakeFlyer Addressed

Reply:
@DonLakeFlyer